src/System.Net.Sockets/tests/FunctionalTests/AcceptAsync.cs

```diff
@@ -8,7 +11,7 @@ namespace System.Net.Sockets.Tests
 {
     public class AcceptAsync
     {
-        private const int TestPortBase = 8000;
+        private const int TestPortBase = 7000;
```

Comment:
Yes: chosen ahead of time. We could also try a port range and make the tests retry on port reuse failure until the range has been exhausted.